airbyte-cdk/python/unit_tests/sources/declarative/stream_slicers/test_datetime_stream_slicer.py

```diff
@@ -454,13 +454,13 @@ def test_request_option(test_name, inject_into, field_name, expected_req_params,
             "%Y-%m-%dT%H:%M:%S.%f%z",
             datetime.datetime(2021, 1, 1, 0, 0, tzinfo=datetime.timezone.utc),
         ),
-        ("test_parse_date_number", "20210101", "%Y%m%d", datetime.datetime(2021, 1, 1, 0, 0, tzinfo=datetime.timezone.utc)),
         (
-            "test_parse_date_datetime",
-            datetime.datetime(2021, 1, 1, 0, 0, tzinfo=datetime.timezone.utc),
-            "%Y%m%d",
+            "test_parse_timestamp",
```

Comment:
add a test for parsing a timestamp to datetime

Reply:
am I reading this correctly that we removed a test case? if so why?

Reply:
@sherifnada yes. datetime.datetime objects are not valid inputs anymore

Reply:
ah makes sense